src/python/grpcio/grpc/__init__.py

```diff
@@ -721,6 +768,26 @@ def peer(self):
     raise NotImplementedError()
 
   @abc.abstractmethod
+  def parent(self, child=None, census=None):
+    """Creates a value to associate invoked RPCs with the current RPC.
+
+    This method's parameters have no guaranteed defaults; if either True or
+    False is desired for the value of an argument the argument should be
+    explicitly passed.
+
+    Args:
+      child: An optional boolean indicating whether or not the invoked RPC
```

Comment:
I think the issue here might be the name of this method and the value that it returns - if parenthood and census propagation are independent, maybe this method should be `ServicerContext.association(child=None, census=None)` and the value it returns be an instance of `Association` and then RPCs get invoked as `my_response = my_stub.MyRpcMethod(my_request, association=my_association)`.

@ctiller, @jtattermusch: what do you think?